config/settings.py

```diff
@@ -155,4 +154,14 @@
     messages.WARNING: "alert-warning",
     messages.ERROR: "alert-danger",
 }
-# END_FEATRUE bootstrap_messages
+# END_FEATURE bootstrap_messages
+
+
+# START_FEATURE django_storages
+DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
```

Comment:
This setting should be inside the `if DEBUG` block.  It should use the local file storage otherwise